### PR TITLE
Django views

### PR DESCRIPTION
**Pull request by Chayapol-c**
create detail and meeting list page.

### CODE REVIEW

File: appointment/models.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ def __str__(self):
         """Return string representative."""
         return self.subject
 
+    @property
     def get_html_url(self):
-        url = reverse("appointment:detail", args=self.id,)
-        return f'<a href="{url}"> {self.title} </a>'
+        url = reverse("appointment:detail", args=(self.id,))
+        return f'<a href="{url}"> {self.subject}</a>'
```
Comment (Chayapol-c) [inline appointment/models.py:25]: I think this could be in other branch.

File: appointment/templates/appointment/detail.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{% extends 'appointment/base.html' %}
```
Comment (Chayapol-c) [inline appointment/templates/appointment/detail.html:1]: add detail.html

File: appointment/utils.py [hunk 3]
```diff
@@ -19,17 +20,18 @@ def formatday(self, day, meetings):
...
+            return f"<td><span class='date'><a href={url}>{day}</span><ul> {meets_in_day} </ul></td>"
         return '<td></td>'
 
     def formatweek(self, theweek, meetings):
         """Return table row HTML tag for each week."""
         week = ''
         for day, weekday in theweek:
             week += self.formatday(day, meetings)
-        return f"<tr> {week} </tr>"
+        return f"<tr><a> {week} </a></tr>"
```
Comment (Chayapol-c) [inline appointment/utils.py]: <a> tag in this line is unnecessary.

File: appointment/views.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 """Views for Django appointment app."""
-from django.shortcuts import render
+from django.shortcuts import render , get_object_or_404, get_list_or_404
```
Comment (Chayapol-c) [inline appointment/views.py]: get_list_or_404 is unused

**Comment by Bouncyyahomie** [comment]
I can't redirect to the meeting list page it's showme this error "Reverse for 'detail' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<meeting_id>[0-9]+)/detail$']"

**Review by nuttapol-kor — APPROVED** [review]
It build passed and I pull your branch to test and it work. Approve!!

File: appointment/templates/appointment/meeting_list.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        <td>{{meet.subject}}</td>
+        <td> location</td>
+        <td>{{meet.start_time|time:"P"}}</td>
+        <td><a href="{% url 'appointment:detail' meet.id%}" class="btn btn-outline-info" role="button">see more</a></td>
+    </tr>
+    {% endfor %}
+</table>
+</div>
+
+{% endblock %}
```
Comment (Bouncyyahomie) [inline appointment/templates/appointment/meeting_list.html:24]: Nice!

**Review by Bouncyyahomie — APPROVED** [review]

**Pull request merged**